### DISCUSSION TITLE
Low volume after update to 20220214

### DISCUSSION THREAD

**Discussion by cYCL157** [rern/rAudio#809] [Fixed Issues]
I updated to 20220214 earlier this afternoon and since then the audio volume is really low.  I have a HiFiberry DAC+, but haven't found anything so far that seems to change what is much lower volume than before the update.

Any help would be much appreciated!

**Comment by rern** [comment]
Try:
- Player > Volume Control - Mixer Device

If still no:
- `alsamixer` > up / down key

**Reply by cYCL157** [reply]
i am not sure what i did, but things seem to be working again.  I did reboot several times and did using the volume control on the mixer device and while it did raise the volume just not the same.  I did reconnect all cables and things seem to work.  I also updated to 22020225 after things were working and all is still great!  thanks for your efforts on this @rern and others -- I really enjoy using this!
